apps/podfetch/config.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "../schema.json",
+  "name": "PodFetch",
+  "port": 8177,
+  "available": true,
+  "exposable": true,
+  "id": "podfetch",
+  "tipi_version": 1,
+  "version": "v2:latest",
+  "categories": ["media"],
+  "description": "A sleek and efficient podcast downloader.",
+  "short_desc": "A sleek and efficient podcast downloader.",
+  "author": "SamTV12345",
+  "source": "https://github.com/SamTV12345/PodFetch",
+  "form_fields": [
+    {
+      "type": "text",
+      "label": "PodFetch Username",
+      "max": 50,
+      "min": 3,
+      "required": true,
+      "env_variable": "PODFETCH_USERNAME"
+    },
+    {
+      "type": "password",
+      "label": "PodFetch Password",
+      "max": 50,
+      "min": 3,
```

Comment:
I think you can increase this to 8, I'm afraid in a future update they require a longer password and thus break existing installations

Reply:
Resolved in 7e5dfa0